Libraries/acdriver/Headers/acdriver/Compile/Output.h

```diff
@@ -41,13 +42,29 @@ class Output {
         Folder,
     };
 
+    enum class ExtraImageType {
```

Comment:
For anything like WEBP that's meant as an extension, I strongly recommend moving that logic into its own files as much as possible.

At the very least, knowledge about WEBP ought to go into its own file.  The ExtraImage* classes also into their own file.

Reply:
I'll move extension logic into it's own file as much as possible :)